tests/integration/targets/lvg_rename/tasks/setup.yml

```
@@ -0,0 +1,50 @@
+---
+# Copyright (c) Contributors to the Ansible project
+# Based on the integraton test for the lvg module
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+- name: Create files to use as disk devices
+  with_sequence: 'count=2'
+  ansible.builtin.command:
+    cmd: "dd if=/dev/zero of={{ remote_tmp_dir }}/img{{ item }} bs=1M count=10"
+    creates: "{{ remote_tmp_dir }}/img{{ item }}"
+
+- name: Show next free loop device
+  ansible.builtin.command:
+    cmd: "losetup -f"
+  changed_when: False
```

YAML booleans should always be lower-case.
```suggestion
  changed_when: false
```

---

My bad, didn't notice this :(